Orange/widgets/model/owneuralnetwork.py

```diff
@@ -306,6 +312,10 @@ def migrate_settings(cls, settings, version):
             if alpha is not None:
                 settings["alpha_index"] = \
                     np.argmin(np.abs(np.array(cls.alphas) - alpha))
+        elif version < 2:
+            alpha = settings.pop("alpha", None)
```

Comment:
Considering the comment below, I'd go for something like:
`settings["alpha_index"] = settings.get("alpha_index", 0) + 1`